sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -8,9 +8,9 @@
 #include "azure/core/internal/log.hpp"
 
 // Private incude
-#include "curl_connection.hpp"
-#include "curl_connection_pool.hpp"
-#include "curl_session.hpp"
+#include "curl_connection_pool_private.hpp"
+#include "curl_connection_private.hpp"
+#include "curl_session_private.hpp"
```

Comment:
headers ending with `_private` will only appear in src/ directory, not in inc/ right?

Reply:
correct.
adding the suffix is only to easily detect if one of this private headers is added to someplace where it shouldn't be.